code/game/objects/items/toys.dm

```diff
@@ -415,6 +415,26 @@
 	playsound(user ? user : src, active ? 'sound/weapons/saberon.ogg' : 'sound/weapons/saberoff.ogg', 20, TRUE)
 	return COMPONENT_NO_DEFAULT_MESSAGE
 
+/obj/item/toy/sword/examine(mob/user)
+	. = ..()
+	. += span_notice("It has a lighting dial currently set to [saber_color] which looks like it can be turned with a <b>screwdriver</b>.")
+
+/obj/item/toy/sword/screwdriver_act(mob/living/user, obj/item/tool)
+	. = TOOLACT_SUCCESSFUL
```

Comment:
Just `return TOOL_ACT_SUCCESSFUL` in the two branches

Reply:
trailing returns suck dick.

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STANDARDS.md#dot-variable-

Reply:
fair enough! at the end of the day same thing, so i'll change it over tomorrow :)